resources/lang/en/messages.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+    'error' => 'Whoops! Something went wrong!',
+    'Delete' => 'Delete',
+    'Task' => 'Task',
+    'Add Task' => 'Add Task',
+    'Current Tasks' => 'Current Tasks',
```

Comment:
```suggestion
    'current_tasks' => 'Current Tasks',
```
Sửa những chỗ bị ảnh hưởng